src/bics_bot/cogs/commands/intro_cmd.py

```diff
@@ -87,6 +95,18 @@ async def intro(
             )
             return
 
+        valid_pattern = r"^(0[1-9]|[12][0-9]|3[01])\.(0[1-9]|1[0-2])\.\d{4}$"
```

Comment:
Consider using dateutils.parser for this. It will parse any date format to a datetime object.

As an example you have this here
```py
from dateutil.parser import parse

print(parse("2016-01-01"))
print(parse("2000/5/05"))
print(parse("2000.05.05"))
print(parse("05/05/2000"))

```